librrd.h

```diff
@@ -126,3 +130,8 @@ int             rrd_del_src(RRD_PLUGIN * plugin, RRD_SOURCE * source);
  * be called every 5 seconds by the client,
  */
 int             rrd_sample(RRD_PLUGIN * plugin);
+
+
+JSON_Value *json_for_source(RRD_SOURCE * source);
+
+JSON_Value *json_for_plugin(RRD_PLUGIN * plugin);
```

Comment:
Again, this is information private to the implementation and must not be exposed to clients.